packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -17,7 +17,7 @@ import {
 } from '@wordpress/components';
 import { Icon, check, published, moreVertical } from '@wordpress/icons';
 import { __, _x } from '@wordpress/i18n';
-import { useSelect } from '@wordpress/data';
+import { useSelect, useDispatch, select } from '@wordpress/data';
```

Comment:
Core code should avoid using global `select` and `dispatch`. Use static selector getter instead.

```js
const { getBlocks } = useSelect( blockEditorStore );
```

Reply:
@Mamaduka - removed the global select and updated as such.

Reply:
@karthick-murugan, sorry for the confusion. I only suggested getting the `getBlocks` selector. You can leave the existing `useSelect` for the block comment ID as it was.

Usually, you'll use this method only to get selectors used in event callbacks like `handleThreadClick`.

Reply:
Thanks @Mamaduka for the clarification. Updated as such.